pulp_2to3_migration/app/viewsets.py

```diff
@@ -94,3 +95,26 @@ class Pulp2ContentViewSet(NamedModelViewSet, mixins.RetrieveModelMixin, mixins.L
     queryset = Pulp2Content.objects.all()
     serializer_class = Pulp2ContentSerializer
     filterset_class = Pulp2ContentFilter
+
+
+class Pulp2to3MappingFilter(BaseFilterSet):
+    """
+    Filter for Pulp2to3Mapping ViewSet.
+    """
+    pulp2_repo_id = filters.CharFilter()
+
+    class Meta:
+        model = Pulp2Repository
+        fields = {
+            'pulp2_repo_id': ['exact', 'in'],
+        }
+
+
+class Pulp2to3MappingViewSet(NamedModelViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin):
+    """
+    ViewSet for Pulp2to3Mapping model.
+    """
+    endpoint_name = 'pulp2to3mapping'
```

Comment:
Let's change this endpoint name to 'pulp2repositories'.

Maybe also change the class names for the viewset and serializer?

Reply:
@goosemania wdyt?

Reply:
+1